browser/brave_rewards/rewards_service_factory.cc

```diff
@@ -62,6 +63,22 @@ RewardsService* RewardsServiceFactory::GetForProfile(
       GetInstance()->GetServiceForBrowserContext(profile, true));
 }
 
+// static
+RewardsService* RewardsServiceFactory::FromBrowserContext(
```

Comment:
this method is not needed and shouldn't be duplicating the logic in `RewardsServiceFactory::GetForProfile` anyway. Everything that is allowed to access this class can also access `Profile`

Reply:
removed